salt/modules/baredoc.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+'''
+Baredoc walks the installed module and state directories and generates
+dictionaries and lists of the function names and their arguments.
+
+.. versionadded:: Neon
+
+'''
+from __future__ import absolute_import, unicode_literals, print_function
+
+# Import python libs
+import logging
+import os
+import re
+
+# Import salt libs
+import salt.loader
+import salt.runner
+import salt.state
+import salt.utils.data
+import salt.utils.files
+import salt.utils.args
+import salt.utils.schema
+
+# Import 3rd-party libs
+from salt.ext import six
+from salt.utils.odict import OrderedDict
+
+log = logging.getLogger(__name__)
+
+
+def _parse_function_definition(fn_def, modulename, ret):
+    args = []
+    match = re.match(r'def\s+(.*?)\((.*)\):$', fn_def)
```

Comment:
Does this handle function definitions that span multiple lines?
```
def foo(
    bar,
    bang):
    pass
``` 

Does this handle function aliases in modules?